CMakeLists.txt

```diff
@@ -218,14 +218,6 @@ else(USE_LAPACK)
   endif()
 endif()
 
-
```

Comment:
This code looks like a c+p error.  Same import is above in this file.

Reply:
I didn't understand this comment?

Reply:
This block https://github.com/apache/incubator-mxnet/blob/master/CMakeLists.txt#L160 seems to be a copy and paste of this block https://github.com/apache/incubator-mxnet/blob/master/CMakeLists.txt#L222 .  Does having it twice serve some purpose, or are we just appending it to the same list twice?